perfkitbenchmarker/linux_benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the Web Search benchmark of Cloudsuite.
+
+More info: http://cloudsuite.ch/websearch/
+"""
+
+import re
+import time
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import docker
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('cloudsuite_web_search_client_heap_size',
+                    '2g',
+                    'Java heap size for Faban client in the usual java format.'
+                    ' Default: 2g.')
```

Comment:
It's not required to specify default because gflags already prints the default in the help string.